scripts/build_version_map.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Directory containing the contracts
+CONTRACTS_DIR="contracts"
+OUTPUT_FILE="./artifacts/version_map.json"
+
+# Create artifacts directory if it doesn't exist
+mkdir -p $(dirname "$OUTPUT_FILE")
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)